cairo/tests/fixtures/runner.py

```diff
@@ -526,7 +526,9 @@ def _factory_rs(entrypoint, *args, **kwargs):
 
         return final_output[0] if len(final_output) == 1 else final_output
 
-    if request.node.get_closest_marker("python_vm"):
-        return _factory_py
-    else:
-        return _factory_rs
+    def _run(entrypoint, *args, vm=None, **kwargs):
+        if vm == "python":
+            return _factory_py(entrypoint, *args, **kwargs)
+        return _factory_rs(entrypoint, *args, **kwargs)
+
+    return _run
```

Comment:
issue with this is that the `RustProgram` may not be defined, see above

```
def rust_program(request, cairo_program: Program) -> RustProgram:
    if request.node.get_closest_marker("python_vm"):
        return None
```

if we want a per test flag, we rather just create two fixtures `cairo_run_py` and `cairo_run(_rs)` instead of the factory I introduced (since actually it's already two fixtures in one)